spec/models/application_letter_spec.rb

```diff
@@ -76,15 +76,17 @@
 
   it "can not be updated if status is changed and application status is locked" do
     application = FactoryGirl.build(:application_letter)
+    application.event.acceptances_have_been_sent = true
+    application.event.rejections_have_been_sent = true
```

Comment:
Title of the test still needs to be renamed.
Also, shouldnt you somehow make sure that `participant_selection_locked` is actually true? Either by manually checking it, or just by using a factory. Because it works with how `participant_selection_locked` is defined atm, but that might change